### PR TITLE
updated forms from fab

### PR DESCRIPTION
**Pull request by srh-sloan**
Forms updated after latest fab export for COF25 and COF25-EOI

### CODE REVIEW

File: fsd_config/form_jsons/cof25/en/community-benefits-cof-25.json [hunk 1]
```diff
@@ -78,7 +60,28 @@
...
+                    "name": "lLCjXj"
+                }
+            ],
+            "next": [
+                {
+                    "path": "/benefits-youll-deliver"
+                }
+            ],
+            "section": "FabDefault",
+            "controller": "start.js"
```
Comment (nuwan-samarasinghe) [inline fsd_config/form_jsons/cof25/en/community-benefits-cof-25.json:84]: I think controller name is `"controller": "./pages/start.js"`

File: fsd_config/form_jsons/cof25/en/environmental-sustainability-cof-25.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
             "title": "Check your answers",
             "components": [],
             "next": [],
+            "section": "ljcxPd",
             "controller": "./pages/summary.js"
```
Comment (nuwan-samarasinghe) [inline fsd_config/form_jsons/cof25/en/environmental-sustainability-cof-25.json:10]: here this is correct

File: fsd_config/form_jsons/cof25/en/inclusiveness-and-integration-cof-25.json [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+                    "name": "GDsdXV"
+                }
+            ],
+            "next": [
+                {
+                    "path": "/how-youll-make-the-asset-inclusive"
+                }
+            ],
+            "section": "FabDefault",
+            "controller": "start.js"
```
Comment (nuwan-samarasinghe) [inline fsd_config/form_jsons/cof25/en/inclusiveness-and-integration-cof-25.json:30]: Incorrect controller name may be we need to compare with old ones ?

**Review by nuwan-samarasinghe — CHANGES_REQUESTED** [review]

File: fsd_config/form_jsons/cof25/en/community-benefits-cof-25.json (hunk 1 above)
Comment (srh-sloan) [reply fsd_config/form_jsons/cof25/en/community-benefits-cof-25.json:84]: It works both ways though

**Review by nuwan-samarasinghe — APPROVED** [review]
LGTM

**Pull request merged**